Kitodo/src/main/webapp/pages/workflowEdit.xhtml

```diff
@@ -36,13 +36,7 @@
     <!--@elvariable id="isViewMode" type="boolean"-->
     <ui:param name="isViewMode"
               value="#{not isEditMode and not isDuplicateMode and not isCreateMode and SecurityAccessController.hasAuthorityToViewWorkflow()}"/>
-
-    <ui:define name="messages">
```

Comment:
This was specifically added in #3558 to prevent the same error message from being displayed twice!

Reply:
But this was presumably leading to #4108 so both solutions aren't really satisfactory....
I'm trying to find a way to fix both...